components/agendor/agendor.app.mjs

```diff
@@ -0,0 +1,243 @@
+import { axios } from "@pipedream/platform";
+
+export default {
+  type: "app",
+  app: "agendor",
+  propDefinitions: {
+    personId: {
+      type: "string",
+      label: "Person ID",
+      description: "The ID of the person to retrieve.",
+      async options(prevContext) {
+        const persons = await this.listPersons(prevContext.page + 1);
+        return persons.data.map((person) => ({
+          label: person.name,
+          value: person.id,
+        }));
+      },
+    },
+    organizationId: {
+      type: "string",
+      label: "Organization ID",
+      description: "The ID of the organization to retrieve.",
+      async options(prevContext) {
+        const organizations = await this.listOrganizations(prevContext.page + 1);
+        return organizations.data.map((organization) => ({
+          label: organization.name,
+          value: organization.id,
+        }));
```

Comment:
```suggestion
        const { data } = await this.listOrganizations(prevContext.page + 1);
        return data.map(({
            name: label, id: value,
        }) => ({
          label,
          value,
        }));
```

Reply:
If you don't mind, I would like to keep it as it is now. I need to break more lines in order to make this destructor because of our lint rules. In the end, it is a little bit harder to read using destructors in this case.
But if you really have a strong opinion about it. I can change it.